Source: Elias Pipping <pipping@exherbo.org>
Upstream: no
Reason: automake 1.12 fails to run
--- gnutls-3.1.6/configure.ac	2013-01-02 19:01:38.000000000 +0100
+++ gnutls-3.1.6.bak/configure.ac	2013-01-04 06:40:53.055761980 +0100
@@ -27,7 +27,7 @@
 
 AM_INIT_AUTOMAKE([1.11.3 no-dist-gzip dist-xz dist-lzip -Wall -Werror -Wno-override])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS([config.h])
 
 AC_MSG_RESULT([***
 *** Checking for compilation programs...
@@ -35,6 +35,7 @@
 
 dnl Checks for programs.
 AC_PROG_CC
+AM_PROG_AR
 AM_PROG_AS
 AC_PROG_CXX
 gl_EARLY
